script/Ciel/Day 7/07_01_CIEL07_1B.txt

```diff
@@ -137,61 +137,62 @@ I haven't forgotten a single thing you said, actually!"
 [sha:6d83565214320233e63cea4ec9e4f5e6e4d8a0bf]{
 -- Page 42, Offset 14908.
 -- 正門前という事を忘れて先輩にくってかかる。
-I lash out at her, forgetting that I'm right in front of the school entrance.
+I lash out at her, forgetting that we're right in front of the school entrance.
 }
 [sha:2fa66e70b9b80eb7835d2e01c4e2541af734952c]{
 -- Page 42, Offset 14909.
 -- 自分でもなんでこんなに気が動転しているのか分からない。
-I have a hard time understanding why I've lost my composure like this.
+I'm not entirely sure why I've lost my composure this badly.
 }
 [sha:b72a7e8c027812a57da92302aa8f6cb70f78e5c7]{
 -- Page 42, Offset 14910.
 -- 先輩がいる事の驚きと、先輩がいる事の喜び。
-Is it the surprise at her being here? Or the happiness?
+I'm both surprised to run into Senpai here, and overjoyed to see her.
 }
 [sha:47062b509c801fa1eb6567b35821a2f291c88ee3]{
 -- Page 42, Offset 14911.
 -- それがまぜこぜになって、マトモに頭が働かなかった。
-My feelings blend together, stopping my head from working properly.
+The combination of those two feelings has sent me completely off the rails.
 }
 [sha:6bf216599a9fc81c43461dde5429c1e34af01591]{
 -- Page 43, Offset 14912.
 -- 「とにかく、どうしてここにいるんですか!?　ぜんぶ終わったんだから、先輩は居ちゃいけないハズでしょう!?」
-"Anyway, why are you here!? Everything's over now, so you shouldn't be around anymore, right?!"
+"Anyway, why are you here?!
+ Everything's over now, aren't you supposed to have cleared out already?!"
 }
 [sha:8355dd27d743f6ecc034d2c5538b557fd5a12bf4]{
 -- Page 44, Offset 14913.
 -- 「なんと。わたしは居てはいけない系の先輩だと？」
-"Oh my, are you saying your Senpai is the sort who shouldn't be around?"
+"Oh my. Are you suggesting that I'm the sort of Senpai you don't want to have around?"
 }
 [sha:cb9f9bc68adf9693f4e598e4fe6501c99832b962]{
 -- Page 44, Offset 14914.
 -- 「そんなワケないでしょう、居てほしい系の先輩に決まってるじゃないですか！　俺をバカにしてるんですか!?」
 "No, of course not!
- Obviously you're the kind of Senpai I'd love to have around!
- You're making fun of me, aren't you?!"
+ You're obviously the sort of Senpai I love to have around!
+ Are you messing with me?!"
 }
 [sha:88247c89fe392767800ec2fd8595bd80d60e9c3c]{
 -- Page 45, Offset 14915.
 -- 「ふむ。この場合、バカにされているのはわたしの方な気がしますが。果たして居ていいのかいけないのか、どっちかに統一していただかないと」
-"Hmm-hmm. In that case, I feel as though I'm the one being made fun of here.
+"Hmm. I feel like %{i}I'm%{/i} the one who's being messed with here.
  Which is it? Can I be around or not?
  I'd rather you be consistent about this."
 }
 [sha:213feb453cd993abe71ab36e486e349170dbdff2]{
 -- Page 46, Offset 14916.
 -- 「いや、だから！
-"No, like I'm saying,
+"Okay, but...!
```

Comment:
I don't quite understand this change

Reply:
I think in the moment I felt like the connection to _what_ he's apparently saying is weak. Like it didn't feel like he established it as enough of a topic beforehand to anchor back to it nicely? I guess the original is fine, though I am going to change the dangling comma still.